data/json/items/comestibles.json

```diff
@@ -20259,5 +20261,99 @@
             "fruit": "mushroom_poison",
             "grow" : 91
         }
-}
+},{
+        "type" : "COMESTIBLE",
+        "id" : "ink_sac",
+        "name" : "ink sac",
+        "weight" : 250,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 0,
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You pop the sac's membrane and smear yourself in it's contents.",
+            "effects" : [
+                {
+                "id": "inked",
+                "duration": 900
+                }
+            ]
+        },
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "MED",
+        "symbol" : "o",
+        "quench" : 0,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 0,
+        "description" : "A translucent organic sac full of black ink. It is known that cephalapod ink has scent masking properties, perhaps you should put this knowledge to use?",
+        "price" : 500,
+        "material" : "null",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "fun" : 0
+},{
+        "type" : "COMESTIBLE",
+        "id" : "tentacle",
+        "name" : "tentacle",
+        "weight" : 500,
+        "color" : "red",
+        "addiction_type" : "none",
+        "spoils_in" : 24,
+        "use_action" : "RAW_MEAT",
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "s",
+        "quench" : 0,
+        "healthy" : -1,
+        "addiction_potential" : 0,
+        "nutrition" : 35,
+        "description" : "A long meaty tentacle. You should probably cook before eating it.",
+        "price" : 500,
+        "material" : "flesh",
+        "tool" : "null",
+        "volume" : 3,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "fun" : -20
+    },{
+        "type" : "COMESTIBLE",
+        "id" : "tentacle_cooked",
+        "name" : "cooked tentacle",
+        "weight" : 250,
+        "color" : "red",
+        "addiction_type" : "none",
+        "spoils_in" : 24,
+        "use_action" : "RAW_MEAT",
+        "stim" : 0,
+        "container" : "null",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "s",
+        "quench" : 0,
+        "healthy" : -1,
+        "addiction_potential" : 0,
+        "nutrition" : 45,
```

Comment:
Tentacles are guaranteed to drop regardless of butchery roll.
For a small animal (say, fully evolved dog) this can almost quadruple the nutrition gain:
`2 pieces of meat * 50 nutrition = 100 nutrition`
`4 tentacles * 45 nutrition * 2 charges = 360 nutrition`